build_tools/cmake/build_all.sh

```diff
@@ -0,0 +1,63 @@
+#!/bin/bash
+
+# Copyright 2019 The IREE Authors
```

Comment:
nit: We can use 2022 for new file?

Reply:
I forked it from an old file. Git doesn't always follow the renames. And we're supposed to preserve the oldest applicable copyright year. I really wish the Apache license hadn't required a copyright year... :-/

Reply:
Following here by accident: I think the Apache license only suggests to add a copyright year but I don‘t think there is a strict requirement.

Reply:
I've talked with Google lawyers about this at unfortunate length. It was only through much complaining that I got our license header as short as it is :-(